### PR TITLE
Added tests for CCP searches

### PR DESCRIPTION
**Pull request by mpelzsherman**
https://github.com/department-of-veterans-affairs/va.gov-team/issues/10448

Enhanced test coverage for CCP searches.

### CODE REVIEW

File: src/applications/facility-locator/tests/e2e/search.cypress.spec.js [hunk 1]
```diff
@@ -83,9 +86,28 @@ describe('Facility search', () => {
 
   it('does not show search result header if no results are found', () => {
     cy.visit('/find-locations?fail=true');
+
+    cy.get('#facility-search-results').should('not.exist');
+  });
+
+  it('finds community dentists', () => {
+    cy.visit('/find-locations');
+
```
Comment (brangi) [inline src/applications/facility-locator/tests/e2e/search.cypress.spec.js:95]: You should mock the correct data and verify

Do something like this:

```    cy.route(
      'GET',
      '/v1/facilities/cpp/',
      'fx:constants/<CCP_MOCK_DATA>',
    ).as('searchCCP');``

Comment (mpelzsherman) [reply src/applications/facility-locator/tests/e2e/search.cypress.spec.js:95]: I was able to use the `MockApi` instead by checking for `window.Cypress` in `src/applications/facility-locator/api/index.js`.

File: src/applications/facility-locator/api/index.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import environment from 'platform/utilities/environment';
 
 // To use vets-api data locally, replace the following with this:
 // export default LiveApi;
-export default (environment.isLocalhost() &&
+export default ((environment.isLocalhost() || window.Cypress) &&
```
Comment (mpelzsherman) [inline src/applications/facility-locator/api/index.js:7]: This change makes it so we always use the MockApi for Cypress tests.

**Review by brangi — APPROVED** [review]
Looking good!

**Pull request merged**